src/lib/config/defaultConfig.js

```diff
@@ -34,7 +34,7 @@ export const APP_CONFIG = {
   SUCCESS_AUTO_DISMISS_SECONDS: 10,
   DEFAULT_CURRENCY: DEFAULT_CURRENCY,
   DEFAULT_LANGUAGE: DEFAULT_LANGUAGE,
-  DEFAULT_RESULTS_SIZE: 15,
+  DEFAULT_RESULTS_SIZE: 100,
```

Comment:
did you check what influence will this change have on the application?

Reply:
Yes, it affects only the patron profile, which is exactly what we want.

Reply:
are you sure? how come all the snapshosts below were modified? You changed the global config, I would expect it affects the application globally

Reply:
I would suggest to check again.
On top of that, the var is called `DEFAULT_RESULTS_SIZE`: you don't want to change a `default` value, normally globally used, for a few specific components.

Reply:
It seems I in fact did some mistake when fetching for uses (maybe I bounded the search or something like that). I have corrected this problem using a new configuration variable.